### PR TITLE
Remove database requests

### PR DESCRIPTION
**Pull request by madcake**

### CODE REVIEW

File: services/controllers/tickers/base.go [hunk 1]
```diff
@@ -65,90 +62,9 @@ func (c Controller) filterTickers(tickers watchmarket.Tickers, rate watchmarket.
...
-			}
-		}
-	}
-	return result, errors.New(watchmarket.ErrNotFound)
-}
-
 func (c Controller) rateToDefaultCurrency(t watchmarket.Ticker, rate watchmarket.Rate) (watchmarket.Rate, bool) {
 	if t.Price.Currency != watchmarket.DefaultCurrency {
-		newRate, err := c.getRateByPriority(strings.ToUpper(t.Price.Currency))
+		newRate, err := c.getCachedRate(strings.ToUpper(t.Price.Currency))
```
Comment (vikmeup) [inline services/controllers/tickers/base.go:67]: Can you rename to getCacheRate()

**Review by vikmeup — APPROVED** [review]
Can you tests against prod to make sure it works identically on different assets?

File: services/controllers/charts/base.go [hunk 2]
```diff
@@ -75,21 +67,8 @@ func (c Controller) HandleChartsRequest(request controllers.ChartRequest) (chart
...
-		if tickers, err = c.getChartsFromMemory(assetData); err != nil {
-			return false
-		}
-	} else {
-		dbTickers, err := c.database.GetTickers([]controllers.Asset{assetData})
-		if err != nil {
-			return false
-		}
-		for _, t := range dbTickers {
-			if t.ShowOption != 2 { // TODO: 2 to constants
```
Comment (vikmeup) [inline services/controllers/charts/base.go:89]: What was this check for?

**Review by hewigovens — COMMENTED** [review]
files in db/models can be removed too?

**Comment by madcake** [comment]
> files in db/models can be removed too?

No, its used in worker

File: services/controllers/info/base.go [hunk 3]
```diff
@@ -84,23 +81,22 @@ func (c Controller) getFromCache(request controllers.DetailsRequest) (controller
...
 
 func (c Controller) getDetailsByPriority(request controllers.DetailsRequest) (controllers.InfoResponse, error) {
-	dbTickers, err := c.database.GetTickers([]controllers.Asset{request.Asset})
-
-	if err != nil || len(dbTickers) == 0 {
-		return controllers.InfoResponse{}, fmt.Errorf("no tickers in db or db error: %w", err)
-	}
-
-	ticker, err := c.getTickerDataAccordingToPriority(dbTickers)
+	key := strings.ToLower(asset.BuildID(request.Asset.CoinId, request.Asset.TokenId))
```
Comment (vikmeup) [inline services/controllers/info/base.go:92]: if key used in two places, let's create a separate function for it

**Review by vikmeup — APPROVED** [review]

**Review by vikmeup — APPROVED** [review]

**Pull request merged**